src/main/java/view/LocationView.java

```diff
@@ -23,17 +25,23 @@ public class LocationView extends JPanel implements ActionListener, PropertyChan
     private final String locationViewName = "Search Locations";
```

Comment:
why did u change location view? u should be working with suggestedlocationview. also im not sure whether we even need the card layout and parent panel for this one; i think it was fine in the previous commit?